types/simpl-schema/index.d.ts

```diff
@@ -132,7 +132,7 @@ interface SimpleSchemaStatic {
   getDefinition(key: string, propList?: any, functionContext?: any): any;
   keyIsInBlackBox(key: string): boolean;
   labels(labels: {[key: string]: string}): void;
-  label(key: any): any;
+  label(key?: string): string | {[key: string]: string};
```

Comment:
should it allow a null? (there is a strict check there):
https://github.com/aldeed/simpl-schema/blob/main/package/lib/SimpleSchema.js#L747